crates/accelerate/src/sampled_exp_val.rs

```diff
@@ -76,14 +76,14 @@ pub fn sampled_expval_complex(
     oper_strs: Vec<String>,
     coeff: PyReadonlyArray1<Complex64>,
     dist: HashMap<String, f64>,
-) -> PyResult<f64> {
+) -> PyResult<Complex64> {
     let coeff_arr = coeff.as_slice()?;
     let out: Complex64 = oper_strs
         .into_iter()
         .enumerate()
         .map(|(idx, string)| coeff_arr[idx] * Complex64::new(bitstring_expval(&dist, string), 0.))
         .sum();
-    Ok(out.re)
```

Comment:
I'm not sure if this change is necessary. It looks like the function was designed to return only the real component. idk how this function is intended to be used. I'm a novice at quantum computing. Enjoying the learning curve!

Reply:
```
struct Complex64 {
    pub re: [f64](https://doc.rust-lang.org/nightly/std/primitive.f64.html),
    pub im: [f64](https://doc.rust-lang.org/nightly/std/primitive.f64.html),
}
```